nengo/spa/actions.py

```diff
@@ -156,6 +158,15 @@ def __init__(self, *args, **kwargs):
         self.args = args
         self.kwargs = kwargs
 
+    def add(self, *args, **kwargs):
+        if self.actions != None:
```

Comment:
Convention is to use `is not` rather than `!=` when comparing to `None`, even in situations where it doesn't really make a difference. Good habit for when it can make a difference.